homeassistant/components/co2signal/sensor.py

```diff
@@ -58,21 +58,18 @@ async def async_setup_entry(
 class CO2Sensor(CoordinatorEntity[CO2SignalCoordinator], SensorEntity):
     """Implementation of the CO2Signal sensor."""
 
-    _attr_state_class = SensorStateClass.MEASUREMENT
+    entity_description: CO2SensorEntityDescription
+    _attr_has_entity_name = True
```

Comment:
I saw that the entity description also sets this field. When we have an entity description is it preferred to use that when possible so everything uses it? or the opposite, only use it when it has something different.

Reply:
I honestly don't know what was added to the entity description 🤷 
Adding it to all descriptions seems unneeded in, for example, this case/PR.